src/app/layout/MainMenu/index.tsx

```diff
@@ -65,11 +65,11 @@ export const MainMenu = ({ ...rest }) => {
   const { isAdmin } = useAccount();
   return (
     <Stack direction="row" spacing="1" {...rest}>
-      <MainMenuItem to="/dashboard">
+      <MainMenuItem to="dashboard">
```

Comment:
This should keep de leading slash no?

Reply:
As it is rendered at the entry point of the router, it is not useful, but in case of a refactor, I agree with you that it is probably better to keep the leading slash.